### DISCUSSION TITLE
Month and year disappears  when prop readonly and minimal in q-date

### DISCUSSION THREAD

**Discussion by zanzara** [quasarframework/quasar#15045] [Ideas / Proposals]
Month and year disappear when using the `readonly` and `minimal` props with the `q-date` component:
```html
 <q-date
  v-model="date"
  readonly
  minimal
/>
```

IMO, only the arrows should disappear.

**Comment by yusufkandemir** [comment]
To extend on this:

https://codepen.io/yusufkandemir/pen/LYrXWRb?editors=101
![image](https://user-images.githubusercontent.com/6266078/205440395-1e1dbe91-a998-41a5-91c4-58af465fa3db.png)

The month and year selector disappearing makes sense in non-minimal + readonly mode as it shows the year and month in the header section. But, in minimal + readonly mode, it's just a loss of important information. You can only see the weekdays. The current behavior might be useful for some use cases where it's only used for display purposes. So, we should add a new prop to support a new behavior. Instead of the month and year selector buttons and arrows, we can just add plain text to render the year and month, looking similar to when the readonly mode is not active, maybe something like this:
![image](https://user-images.githubusercontent.com/6266078/205440653-f2eafee1-870f-43a1-b06a-c1253f2d09e2.png)

**Comment by zanzara** [comment]
In the meantime, until such FR would be available, one can do some CSS overwrites:
```html
<style lang="scss" scoped>
:deep(.q-date__navigation) {
  pointer-events: none;
  .q-date__arrow {
    display: none;
  }
}
:deep(.q-date__header-subtitle) {
  pointer-events: none;
}
</style>
```
